src/engine/source/indexerconnector/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ add_library(indexerconnector::iindexerconnector ALIAS indexerconnector_iindexerc
 
 # Include directories
 include_directories(include)
+include_directories(${CMAKE_SOURCE_DIR}/source/base/include/base/utils/)
```

Comment:
remove this, we have a target for the base lib.

Reply:
done